ci/nightly/pipeline.yml

```diff
@@ -140,6 +141,17 @@ steps:
           composition: testdrive
           args: [--aws-region=us-east-2, --workers=1]
 
+  - id: testdrive-workers-32
+    label: ":racing_car: testdrive with --workers 32"
+    timeout_in_minutes: 30
+    plugins:
+      - ./ci/plugins/scratch-aws-access: ~
+      - ./ci/plugins/mzcompose:
+          composition: testdrive
+          args: [--aws-region=us-east-2, --workers=32]
+    agents:
+      queue: x86-64-large
```

Comment:
Ok, sorry for the whiplash. This should be good to go if you revert to the original spelling of `x86_64-large`.